app/src/main/java/hu/vmiklos/plees_tracker/SleepsAdapter.kt

```diff
@@ -84,6 +84,12 @@ class SleepsAdapter(
         val durationMS = sleep.stop - sleep.start
         val durationText = DataModel.formatDuration(durationMS / 1000)
         holder.duration.text = durationText
+        val nextSleepReferenceTime =
+                if (position == 0) System.currentTimeMillis()
+                else data[position - 1].start
```

Comment:
Are inline if expressions ok with your code-style or do you prefer adding curly braces and line breaks?
I can't quite remember why exactly i chose this style back when i wrote this and whether i saw something similar elswehere in the code or from whatever reference i used to achieve a Java ternary operator equivalent in Kotlin.

```suggestion
        val nextSleepReferenceTime = if (position == 0) {
                    System.currentTimeMillis()
                } else {
                    data[position - 1].start
                }
```

Reply:
I'm fine either way, but if I have to choose, I like the suggested change. The reason is that this way it's easier to inject debug printfs to existing code.